### DISCUSSION TITLE
How to install on ubuntu 24.04?

### DISCUSSION THREAD

**Discussion by drshajul** [cockpit-project/cockpit-files#709] [Q&A] [answered]
I tried `make devel-install`    and `sudo make install`, but nothing seems to work.

**Comment by martinpitt** [comment, marked as answer]
You need to be a bit more specific. Did these commands fail or succeed? If they failed, what was the output? What does `cockpit-bridge --packages` show? And most importantly, what is your cockpit-bridge version? (`dpkg -l cockpit-bridge`). Note that you need  a recent version from noble-backports, 314 from the release isn't sufficient. (That's the most likely explanation)

**Reply by drshajul** [reply]
Thanks. Yes, version was the problem.

**Comment by Jiv-au** [comment]
Thanks for the answer @martinpitt.

My system is Ubuntu 24.04LTS:

`# sudo apt list --installed | grep cockpit-bridge`
`cockpit-bridge/noble,now  314-1 amd64 [installed,automatic]`

Followed the recommendation in [Cockpit Project install page](https://cockpit-project.org/running.html) to resolve this:
> We recommend installing or updating the latest version from backports.

`. /etc/os-release`
`sudo apt install -t ${VERSION_CODENAME}-backports cockpit`

After installing the Cockpit backport for 24.04LTS (above commands) the version is now:

`cockpit-bridge/noble-backports,now 339-1~bpo24.04.1 all [installed,automatic]`

And cockpit-files started working.

**Comment by martinpitt** [comment]
Note that it's finally in official Ubuntu 24.04 LTS backports! https://launchpad.net/ubuntu/+source/cockpit-files/36-1~bpo24.04.1
